update_leggo_data.sh

```diff
@@ -107,6 +108,48 @@ check_errs $? "Não foi possível baixar dados de comissões."
 
 }
 
+generate_backup(){
+
+pprint "Gerando backup dos csvs"
+       mkdir -p ${BACKUP_FOLDERPATH}${backup_file}
+       docker run -d --rm -it --name alpine --mount type=volume,source=leggo_data,target=/data alpine
+       list_csv=( 
+       alpine:/data/camara 
+       alpine:/data/senado 
+       alpine:/data/pops 
+       alpine:/data/proposicoes.csv 
+       alpine:/data/coautorias_edges.csv 
+       alpine:/data/coautorias_nodes.csv 
+       alpine:/data/trams.csv 
+       alpine:/data/hists_temperatura.csv 
+       alpine:/data/autorias.csv 
+       alpine:/data/pautas.csv 
+       alpine:/data/progressos.csv 
+       alpine:/data/emendas.csv 
+       alpine:/data/atuacao.csv 
+       alpine:/data/comissoes.csv 
+       alpine:/data/pressao.csv 
+       alpine:/data/anotacoes_especificas.csv 
+       alpine:/data/interesses.csv 
+       alpine:/data/anotacoes_gerais.csv 
+       alpine:/data/entidades.csv 
+       alpine:/data/autores_leggo.csv 
+       )
+       for index in ${list_csv[@]}; do 
+              docker cp $index ${BACKUP_FOLDERPATH}${backup_file}
+       done
```

Comment:
```suggestion
       done
       docker stop alpine
```

Reply:
Se o container não for parado depois de cada execução, na próxima execução um erro na execução do container irá acontecer.